charts/curiefense/curiefense/1.0.1+up1.0.0/questions.yaml

```diff
@@ -0,0 +1,39 @@
+questions:
+  - variable: secret.file
+    description: "You need to have access to AWS Credentials in order to create the file s3cfg.yaml in the istio-helm folder'."
+    required: true
+    type: string
+    default: cluster-configuration-secret
+    label: Cluster Configuration
+    group: "Cluster Configuration"
+
+questions:
+  - variable: aws.credentials.conversion
+    description: "You need to convert your AWS credentials with these commands cat << EOF | base64 -w0 [default] access_key = xx secret_key = xx EOF"
+    required: true
+    type: string
+    default: cluster-configuration-secret
+    label: Cluster Configuration
+    group: "Cluster Configuration"
+
+  - variable: helm.version
```

Comment:
I see some of the values being set by these questions are not in the chart's values.yaml, which is fine if intentional, but I don't see them being used in any of the templates either (For example this value `helm.version`).